src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -20229,9 +20229,15 @@ public static bool TakeOutParam<T>(T y, out T x)
 
                 var tree = compilation.SyntaxTrees.Single();
                 Assert.Empty(GetOutVarDeclarations(tree));
+                AssertNoGlobalStatements(tree);
```

Comment:
Parser doesn't parse anything as a global statement, unless it is in a Script mode.